src/index.tsx

```diff
@@ -4,11 +4,14 @@
  * Copyright 2016-Present Datadog, Inc.
  */
 
-import { DdSdkConfiguration } from './types';
```

Comment:
I didn't know that this file is also generated. This diff is wrong then and we need to update generator.